frontend/messages/fr.json

```
@@ -2697,6 +2697,59 @@
+	"people": "Personnes",
+	"lifecycle": "Cycle de vie",
+	"comingSoon": "Bientôt disponible",
+	"quality": "Qualité",
+	"benefits": "Bénéfices",
+	"subProjects": "Sous-projets",
+	"noSubProjects": "Aucun sous-projet",
+	"noResponsibilityMatrices": "Aucune matrice de responsabilités",
+	"financials": "Finances",
+	"expectedBudget": "Budget prévu",
+	"actualCost": "Coût engagé",
+	"remaining": "Reste à engager",
```

Biome (2.4.15)

[error] The key remaining was already declared.

(lint/suspicious/noDuplicateObjectKeys)